### PR TITLE
Add anti mt manager

### PR DESCRIPTION
**Pull request by qwq233**
Signed-off-by: qwq233 <qwq233@qwq2333.top>

# Title
Add anti mt manager

## 介绍 / Description
Add anti mt manager

## 更改内容 / Types of Changes
- 增强优化 Enhancement/optimization

## 检查列表 / Checklist
- [x] tested
- [x] works on my device

### CODE REVIEW

**Comment by qwq233** [comment]
run tests.

**Review by KyuubiRan — DISMISSED** [review]

**Comment by qwq233** [comment]
run tests

**Comment by qwq233** [comment]
run tests.

**Comment by qwq233** [comment]
run tests.

File: app/src/main/cpp/Natives.cpp [hunk 1]
```diff
@@ -0,0 +1,564 @@
...
+
+//jint JNI_OnLoad(JavaVM *jvm, void *resv) {
+//}
+
+/*
+ * Class:     nil_nadph_qnotified_util_Natives
+ * Method:    mwrite
+ * Signature: (JI[BI)V
+ */
+EXPORT void Java_nil_nadph_qnotified_util_Natives_mwrite
```
Comment (keta1) [inline app/src/main/cpp/Natives.cpp:25]: 这堆内存读写的代码，不需要

File: app/src/main/cpp/Natives.cpp [hunk 2]
```diff
@@ -0,0 +1,564 @@
...
+            success = false;
+            break;
+        }
+    }
+    free(types);
+    free(uins);
+    return success;
+}
+
+jboolean handleSendCardMsg(JNIEnv *env, jclass clazz, jobject rt, jobject session, jstring msg) {
```
Comment (keta1) [inline app/src/main/cpp/Natives.cpp:436]: 这发卡片的代码更没用啊

**Review by keta1 — CHANGES_REQUESTED** [review]

File: app/src/main/cpp/Natives.cpp (hunk 2 above)
Comment (qwq233) [reply app/src/main/cpp/Natives.cpp:436]: 哦 那是忘了删

**Pull request closed without merging**